src/Microsoft.AspNet.Razor/Parser/SyntaxConstants.cs

```diff
@@ -27,6 +27,7 @@ public static class CSharp
             public static readonly string ElseIfKeyword = "else if";
             public static readonly string NamespaceKeyword = "namespace";
             public static readonly string ClassKeyword = "class";
+            public static readonly string HelperKeyword = "helper";
```

Comment:
Separate this from the rest and leave a comment above it stating that it's only for error use-cases.